app/models/user.rb

```diff
@@ -83,9 +83,7 @@ def recommended_projects(episode = nil)
 
   def self.for_ichain_username(username, attributes)
     user = find_by(name: username)
-    if user
-      user.update_attributes(email: attributes[:email]) if user.email != attributes[:email]
-    else
```

Comment:
I am not sure about this change. I guess the idea here is to keep the user's email address up to date for authentications that rely on external services like LDAP.

Something the other hackweek hackers should double check